models/addresses_events/ethereum/addresses_events_ethereum_schema.yml

```diff
@@ -10,6 +10,17 @@ models:
     config:
       tags: ['table', 'funded', 'addresses_events', 'ethereum']
     description: "Table showing who first funded each Ethereum address in ETH"
+    tests:
+      - check_seed:
+          seed_file: ref('first_funded_by_seed')
+          filter:
+            blockchain: ethereum
+          match_columns:
+            - address
+          check_columns:
+            - first_funded_by
+            - block_number
+            - tx_hash
     columns:
```

Comment:
after typo fix in models, can also add `blockchain` in all schema files to keep documentation clean